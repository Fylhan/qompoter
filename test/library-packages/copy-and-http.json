{
	"name": "install-action-for-library-packages/copy-and-http",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/acme": "v1.*-lib",
		"fylhan/qompoter-test-package4git": "v1.0-lib",
		"fylhan/qompoter-test-package": "v1.0-lib"
	},
	"repositories": {
		"fylhan/qompoter-test-package4git": { "url": "https://github.com/Fylhan/qompoter-test-package4git/releases/download/v1.0/qompoter-test-package4git-v1.0-lib.tar.gz" },
		"fylhan/qompoter-test-package": { "url": "https://wrong-uri.tar.gz" }
	}
}
