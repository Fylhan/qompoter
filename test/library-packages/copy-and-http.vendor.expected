vendor
vendor/acme
vendor/acme/qompoter.json
vendor/acme/qompoter.pri
vendor/lib_linux_64
vendor/lib_linux_64/acme_fake.a
vendor/lib_linux_64/qompoter-test-package_fake.a
vendor/lib_linux_64/qompoter-test-package4git_fake.a
vendor/qompoter-test-package
vendor/qompoter-test-package/qompoter.json
vendor/qompoter-test-package/README.md
vendor/qompoter-test-package4git
vendor/qompoter-test-package4git/qompoter.json
vendor/qompoter-test-package4git/README.md
vendor/qompote.pri
vendor/vendor.pri
